Doc/whatsnew/3.7.rst

```diff
@@ -239,6 +239,13 @@ pdb
 argument.  If given, this is printed to the console just before debugging
 begins.
 
+re
+--
+
+The flags :const:`re.ASCII`, :const:`re.LOCALE` and :const:`re.UNICODE`
+can be set for the part of a regular expression.
```

Comment:
How about "... can be used as inline regular expression flags."  ...?

Reply:
They can be used as inline regular expression flags before, but affect the whole expression.

Reply:
How about "...can be used within the scope of a group"?